balboa-http/src/main/scala/com/socrata/balboa/server/MetricsServlet.scala

```diff
@@ -26,31 +30,48 @@ object MetricsRest {
   val FieldKey = "field"
 
   val seriesMeter = com.yammer.metrics.Metrics.newTimer(
-    classOf[MetricsRest], "series queries", TimeUnit.MILLISECONDS, TimeUnit.SECONDS)
+    classOf[MetricsServlet], "series queries", TimeUnit.MILLISECONDS, TimeUnit.SECONDS)
 
   val rangeMeter = com.yammer.metrics.Metrics.newTimer(
-    classOf[MetricsRest], "range queries", TimeUnit.MILLISECONDS, TimeUnit.SECONDS)
+    classOf[MetricsServlet], "range queries", TimeUnit.MILLISECONDS, TimeUnit.SECONDS)
 
   val periodMeter = com.yammer.metrics.Metrics.newTimer(
-    classOf[MetricsRest], "period queries", TimeUnit.MILLISECONDS, TimeUnit.SECONDS)
+    classOf[MetricsServlet], "period queries", TimeUnit.MILLISECONDS, TimeUnit.SECONDS)
 
+  // Match paths like /metrics/:entityId and /metrics/:entityId/whatever
+  get("""^\/([^\/]+).*""".r)(getMetrics)
+  def getMetrics(): Any = {
+    val entityId = params("captures")
 
-  def get(entityId: String, params: Params, accepts: Seq[String]): ResponseWithType = {
     val period = params.get(PeriodKey).map(Extractable[Period].extract) match {
       case Some(Right(value)) => value
-      case Some(Left(err)) => return badRequest(PeriodKey, err)
-      case None => return required(PeriodKey)
+      case Some(Left(err)) =>
+        contentType = json
+        return badRequest(PeriodKey, err).result
```

Comment:
@jacobsi These `.result` field accesses, as well as the `contentType = x` will all disappear when we remove the Protobuf API.